iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/extensions/VoltageRegulationSerDe.java

```diff
@@ -8,91 +8,75 @@
-                        .put("1.10", "http://www.powsybl.org/schema/iidm/ext/voltage_regulation/1_10")
-                        .put("1.11", "http://www.powsybl.org/schema/iidm/ext/voltage_regulation/1_11")
-                        .put("1.12", "http://www.powsybl.org/schema/iidm/ext/voltage_regulation/1_12")
-                        .build());
+        @Override
+        public VersionInfo getVersionInfo() {
+            return versionInfo;
+        }
     }
 
-    private static ImmutableSortedSet<String> reversedNaturalOrderOf(String... versions) {
-        return ImmutableSortedSet.<String>reverseOrder().add(versions).build();
+    public VoltageRegulationSerDe() {
+        super(VoltageRegulation.NAME, VoltageRegulation.class, Version.values());
     }
 
     @Override
     public InputStream getXsdAsStream() {
-        return getClass().getResourceAsStream("/xsd/voltageRegulation_V1_1.xsd");
-    }
-
-    @Override
-    public List<InputStream> getXsdAsStreamList() {
-        return List.of(Objects.requireNonNull(getClass().getResourceAsStream("/xsd/voltageRegulation_V1_0.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/voltageRegulation_V1_0_legacy.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/voltageRegulation_V1_1.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_2.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_3.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_4.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_5.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_6.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_7.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_8.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_9.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_10.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_11.xsd")),
-                Objects.requireNonNull(getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_12.xsd")));
+        return getClass().getResourceAsStream("/xsd/compatibility/voltage_regulation/voltageRegulation_V1_1.xsd");
```

Comment:
This file is not in the "compatibility/voltage_regulation" sub-directory:
```suggestion
        return getClass().getResourceAsStream("/xsd/voltageRegulation_V1_1.xsd");
```